actioncable/lib/action_cable/test_helper.rb

```diff
@@ -129,5 +129,14 @@ def pubsub_adapter # :nodoc:
       def broadcasts_size(channel)
         broadcasts(channel).size
       end
+
+      def error_message(data, stream, new_messages)
```

Comment:
Test helpers are included in all test calsses, and this `error_message` method is quite generic, I fear it may conflict.

It's only called in one place so I think I'd inline it.

Reply:
you're right. 
I'm going to change that.

Reply:
@byroot fixed